module/gui/render_gui.py

```diff
@@ -0,0 +1,40 @@
+import PySimpleGUI as sg
```

Comment:
The main of the GUI is to move completely from the cli (to make it a stand alone). But in this situation it still rely on the cli to make the download....Anyways i like the simplicity of the dashboard 👍

Reply:
It is suppose to include the download mode which is not taken at the initial start of the app